### PR TITLE
Contributing info

### PR DESCRIPTION
**Pull request by markemeis**
Details on contributing to Arete

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Contributing to Arete
+
+Welcome! We're excited that you're interested in contributing. 
+
+Below you will find some basic steps required to be able to contribute to the project. If you have any questions about this process or any other aspect of contributing to a Yipee open source project, feel free to send an email to support@yipee.io and we'll get your questions answered as quickly as we can.
+
+Pull Requests are always welcome, however they will only be accepted if they provide a reasonably test coverage.
```
Comment (jrryjcksn) [inline CONTRIBUTING.md]: "provide reasonable test coverage"

File: reporting_bugs.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Reporting bugs
+
+If any part of the etcd project has bugs or documentation mistakes, please let us know by [opening an issue](https://github.com/yipeeio/arete/issues). We treat bugs and mistakes very seriously and believe no issue is too small. Before creating a bug report, please check that an issue reporting the same problem does not already exist.
```
Comment (jrryjcksn) [inline reporting_bugs.md]: ^etcd^arete

File: reporting_bugs.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+To make the bug report accurate and easy to understand, please try to create bug reports that are:
+
+- Specific. Include as much details as possible: which version, what environment, what configuration, etc. If the bug is related to running the etcd server, please attach the etcd log (the starting log with etcd configuration is especially important).
```
Comment (jrryjcksn) [inline reporting_bugs.md]: Scratch the etcd server discussion

File: reporting_bugs.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+- Reproducible. Include the steps to reproduce the problem. We understand some issues might be hard to reproduce, please includes the steps that might lead to the problem. If possible, please attach the affected etcd data dir and stack strace to the bug report.
```
Comment (jrryjcksn) [inline reporting_bugs.md]: etcd...

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+# Contributing to Arete
+
+Welcome! We're excited that you're interested in contributing. 
+
+Below you will find some basic steps required to be able to contribute to the project. If you have any questions about this process or any other aspect of contributing to a Yipee open source project, feel free to send an email to support@yipee.io and we'll get your questions answered as quickly as we can.
```
Comment (D3nn) [inline CONTRIBUTING.md]: Just checking, I thought you were planning on pulling the plug on the yipee.io gsuite account.  Also, we're likely to lose our amazon account in the not-too-distant future, and that will take our yipee.io domain with it.

Do we want to see if we can get that domain transferred to us?

File: reporting_bugs.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+- Isolated. Please try to isolate and reproduce the bug with minimum dependencies. It would significantly slow down the speed to fix a bug if too many dependencies are involved in a bug report. Debugging external systems that rely on etcd is out of scope, but we are happy to provide guidance in the right direction or help with using etcd itself.
```
Comment (jrryjcksn) [inline reporting_bugs.md]: etcd...

File: CONTRIBUTING.md (hunk 5 above)
Comment (jd3quist) [reply CONTRIBUTING.md]: 👍 Same comment from me

Comment (markemeis) [reply CONTRIBUTING.md]: I thought the same thing.  I'm going to change it to say open an issue.

File: CONTRIBUTING.md (hunk 1 above)
Comment (markemeis) [reply CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    The simplest way to accomplish this is to add `-s` or `--signoff` to your `git commit` command.
+
+    You must use your real name (sorry, no pseudonyms, and no anonymous contributions).
+
+
+Additional information on DCOs located at https://github.com/probot/dco#how-it-works
+
+## Reporting bugs and creating issues
+
+Reporting bugs is one of the best ways to contribute. However, a good bug report has some very specific qualities, so please read over our short document on [reporting bugs](https://github.com/etcd-io/etcd/blob/master/Documentation/reporting_bugs.md) before submitting a bug report. This document might contain links to known issues, another good reason to take a look there before reporting a bug.
```
[the remark this answers is not in the document]
Comment (markemeis) [reply CONTRIBUTING.md]: fixed.

File: reporting_bugs.md (hunk 3 above)
Comment (markemeis) [reply reporting_bugs.md]: removed.

File: reporting_bugs.md (hunk 4 above)
Comment (markemeis) [reply reporting_bugs.md]: removed.

File: reporting_bugs.md (hunk 6 above)
Comment (markemeis) [reply reporting_bugs.md]: updated to reference arete instead.

File: reporting_bugs.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (markemeis) [reply reporting_bugs.md]: removed.

File: reporting_bugs.md (hunk 2 above)
Comment (D3nn) [inline reporting_bugs.md]: Still a reference to etcd in the first line of this section.

File: DCO.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
+By making a contribution to this project, I certify that:
+
+1. The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or
+
+1. The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or
+
+1. The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.
```
[the remark this answers is not in the document]
Comment (markemeis) [reply DCO.md]: Good catch - this was straight from several sources. I've changed to be the actual text from https://developercertificate.org

File: DCO.md [hunk 9]
```diff
@@ -1,12 +1,40 @@
...
+    of my knowledge, is covered under an appropriate open source
+    license and I have the right under that license to submit that
+    work with modifications, whether created in whole or in part
+    by me, under the same open source license (unless I am
+    permitted to submit under a different license), as indicated
+    in the file; or
 
-1. The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.
+(c) The contribution was provided directly to me by some other
+    person who certified (a), (b) or (c) and I have not modified
```
[the remark this answers is not in the document]
Comment (markemeis) [reply DCO.md]: I believe that the person committing the code is assuming responsibility for the changes.

**Pull request merged**